game.py

```diff
@@ -0,0 +1,9 @@
+import tkinter as tk
+from piece import *
+from tile import *
+from Appp import *
```

Comment:
I think you used `Appp` to prevent conflicts with its class, but you have nothing to worry about. As an alternative way you can rename the file to `main.py` instead